api/src/main/java/org/apache/iceberg/SnapshotRefType.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Locale;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 
-enum SnapshotRefType {
+public enum SnapshotRefType {
```

Comment:
I haven't followed the initial implementation but it seems `SnapshotRef` class offers helper boolean methods to check if a ref is a branch or is a tag and hides `SnapshotRefType` on purpose.

Is there a particular reason to make this public, @snazy?

Reply:
Agreed. I don't see a reason to make this public.